app/views/content/blog/the-head-of-science-inspiring-students-in-blackpool.md

```diff
@@ -0,0 +1,46 @@
+---
+title: The Head of Science inspiring students in Blackpool
+date: "2022-02-09"
+author: Jessica Walmsley
+images:
+an_image:
+  path: "media/images/content/blog/jessica-walmsley.jpg"
+  thumbnail_path: "media/images/content/blog/thumbnails/jessica.jpg"
+  alt: "Jessica Walmsley against a plain background"
```

Comment:
```suggestion
  jessica-walmsley:
    path: "media/images/content/blog/jessica-walmsley.jpg"
    thumbnail_path: "media/images/content/blog/thumbnails/jessica.jpg"
```

@emilytraff slight tweak needed here, the placeholder `$jessica-walmsley$` below needs to match the value above.